packages/patterns/package.json

```diff
@@ -0,0 +1,139 @@
+{
+  "name": "@carbon/ibmdotcom-patterns",
```

Comment:
One thought on the name... Should we add `-react` to the end leaving the door open for vanilla, angular and vue if we get the time and resources?

Reply:
we probably will need to at minimum create the vanilla version eventually. was thinking that name would be `@carbon/ibmdotcom-patterns-vanilla`. We could rename this one to `@carbon/ibmdotcom-patterns-react` or keep it as `@carbon/ibmdotcom-patterns` if react is going to be the "source of truth"? I'm fine either way.
Thoughts @kennylam @annawen1 ?